articles/logic-apps/quickstart-create-first-logic-app-workflow.md

```diff
@@ -15,52 +15,52 @@ ms.date: 10/20/2019
 
 # Quickstart: Create your first automated workflow with Azure Logic Apps - Azure portal
 
-This quickstart introduces how to build your first automated workflow with [Azure Logic Apps](../logic-apps/logic-apps-overview.md). In this article, you create a logic app that regularly checks a website's RSS feed for new items. If new items exist, the logic app sends an email for each item. When you're done, your logic app looks like this workflow at a high level:
+This quickstart introduces how to build your first automated workflow with [Azure Logic Apps](../logic-apps/logic-apps-overview.md). In this article, you create a Logic App that regularly checks a website's RSS feed for new items. If new items exist, the Logic App sends an email for each item. When you're done, your Logic App looks like this workflow at a high level:
 
-![High-level example logic app workflow](./media/quickstart-create-first-logic-app-workflow/quickstart-workflow-overview.png)
+![High-level example Logic App workflow](./media/quickstart-create-first-logic-app-workflow/quickstart-workflow-overview.png)
 
-To follow this quickstart, you need an email account from a provider that's supported by Logic Apps, such as Office 365 Outlook, Outlook.com, or Gmail. For other providers, [review the connectors list here](https://docs.microsoft.com/connectors/). This logic app uses an Office 365 Outlook account. If you use another email account, the overall steps are the same, but your UI might slightly differ.
+To follow this quickstart, you need an email account from a provider that's supported by Logic Apps, such as Office 365 Outlook, Outlook.com, or Gmail. For other providers, [review the connectors list here](https://docs.microsoft.com/connectors/). This Logic App uses an Office 365 Outlook account. If you use another email account, the overall steps are the same, but your UI might slightly differ.
 
 Also, if you don't have an Azure subscription, [sign up for a free Azure account](https://azure.microsoft.com/free/).
 
 ## Sign in to the Azure portal
 
 Sign in to the [Azure portal](https://portal.azure.com) with your Azure account credentials.
 
-## Create your logic app
+## Create your Logic App
 
 1. From the Azure home page, in the search box, find and select **Logic Apps**.
 
    ![Find and select "Logic Apps"](./media/quickstart-create-first-logic-app-workflow/find-select-logic-apps.png)
 
 1. On the **Logic Apps** page, select **Add**.
 
-   ![Add new logic app](./media/quickstart-create-first-logic-app-workflow/add-new-logic-app.png)
+   ![Add new Logic App](./media/quickstart-create-first-logic-app-workflow/add-new-logic-app.png)
 
-1. Under **Create logic app**, provide details about your logic app as shown here. After you're done, select **Create**.
+1. Under **Create Logic App**, provide details about your Logic App as shown below. After you are done, select **Create**.
```

Comment:
- **Create logic app** in bold appears as-is in the portal, so please revert.
- "you're" is OK, so please revert.